src/pages/gateway/fastly.md

```diff
@@ -0,0 +1,251 @@
+     - **Name** - api_mesh_miss
+     - **Type** - **miss**
+     - **Priority** - **60**
+     - **Content**:
+
+        ```vcl
+        if (req.url ~ "^/api/") {
+        set req.backend = F_graph_prod_adobe_io;
+        }
+        //apiMESH prod mapping
+        if (req.url ~ "^/api/<mesh_id>") {
+        set bereq.http.tenantuuid = "<mesh_id>";
+        set bereq.http.x-api-key = "<mesh_api_key>";
+        # Optionally add another environment
+        # }
+        # //apiMESH stage mapping
+        # if (req.url ~ "^/api/<mesh_id>") {
+        # set bereq.http.tenantuuid = "<mesh_id>";
+        # set bereq.http.x-api-key = "<mesh_api_key>";
+        # }
+        ```
+
+    The remaining snippets allow you to query your mesh URL without appending the `api_key`:
+
+   - Cache pass - replace the `<mesh_id>` and `<mesh_api_key>` placeholders with the information from your mesh URL, which has the following structure: `https://graph.adobe.io/api/<meshId>/graphql?api_key=<mesh_api_key>`:
+     - **Name** - api_mesh_pass
+     - **Type** - **pass**
+     - **Priority** - **10**
+     - **Content**:
+
+        ```vcl
+        if (req.url ~ "^/api/") {
+        set req.backend = F_graph_prod_adobe_io;
+        }
+        //apiMESH prod mapping
+        if (req.url ~ "^/api/<mesh_id>") {
+        set bereq.http.tenantuuid = "<mesh_id>";
+        set bereq.http.x-api-key = "<mesh_api_key>";
+        }
+        # Optionally add another environment
+        # //apiMESH stage mapping
+        # if (req.url ~ "^/api/<mesh_id>") {
+        # set bereq.http.tenantuuid = "<mesh_id>";
+        # set bereq.http.x-api-key = "<mesh_api_key>";
+        # }
+        ```
+
+   - Cache fetch:
+     - **Name** - api_mesh_fetch
+     - **Type** - **fetch**
+     - **Priority** - **10**
+     - **Content**:
+
+        ```vcl
+        if (req.http.x-commerce-bypass-fastly-cache == "true") {return(pass);}
+        ```
+
+   - Cache deliver:
+     - **Name** - api_mesh_deliver
+     - **Type** - **deliver**
+     - **Priority** - **10**
+     - **Content**:
+
+        ```vcl
+        if (req.http.x-commerce-bypass-fastly-cache == "true") {return(deliver);}
+        ```
+
+1. In **Fastly Configuration** > **Backend Settings** click **Create**. Add a new backend with the following information:
+
+   - **Condition** - `req.url ~ "^/api/(.*)" ,`
+   - **Address** - **graph.adobe.io**
+   - **Priority** - **8**
+
+1. In **Fastly Configuration** click **Upload VCL to Fastly**. Click **Save Config**.
+
+## Configure headers in API Mesh
+
+<InlineAlert variant="info" slots="text"/>
+
+After adding VCL snippets in the [Fastly Setup](#configure-fastly-in-adobe-commerce), your Commerce GraphQL URL is now in the following format with no `api_key` appended: `https://graph.adobe.io/api/<meshId>/graphql`
+
+To distinguish between requests from users and requests from API Mesh, use the following configuration option to prevent Fastly from caching headers that come directly from API Mesh:
```

Comment:
Perhaps mention that this is a source operationHeader to help the user understand where to add it